specification.markdown

```diff
@@ -535,6 +535,40 @@ An `<element>` SHALL NOT have more than one `<implementation_note>` of the same
 
 The note\_attribute attribute is REQUIRED.
 
+#### \<implementation_note> Example
+
+The following fragment of an EBML Schema demonstrates how an `<implementation_note>` is used. In this case an EBML Schema documents a list of items that are described with an optional cost. The Currency Element uses an `<implementation_note>` to say that the Currency Element is REQUIRED if the Cost Element is set, otherwise not.
+
+```xml
+<element name="Item" path="1*1(\Items\)" id="0x4025" type="master">
```

Comment:
There shouldn't be a `\` at the end of the path.

Reply:
`minOccurs` 1 is not the default value so should be set in Schema.
`maxOccurs` 1 is not the default value so should be set in Schema.